### PR TITLE
table focus

### PR DESCRIPTION
**Pull request by helen-v**
## Description
https://jira-eng-sjc12.cisco.com/jira/browse/CAX-925

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/table-advanced/TableAdvanced.ts [hunk 1]
```diff
@@ -58,6 +59,7 @@ export namespace TableAdvanced {
 
     private dragover = (e: Event) => e.preventDefault();
 
+    @queryAll("tr[tabindex]") tableRaws?: HTMLTableRowElement[];
```
Comment (futurethang) [inline web-components/src/components/table-advanced/TableAdvanced.ts:62]: spelling: did you mean `tableRows`?

**Comment by futurethang** [comment]
Based on the ticket I see we need a more roving-tab-index solution like detailed in https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html
Which is maybe a little different than what CAX-925 asks for. Difference being, should arrow keys navigate `up/down/left/right` to reach every cell (like that ARIA docs state) or just `up/down` like Shivam specifies on the ticket.

My feeling is align to the full ARIA nav capability instead of just row by row

File: web-components/src/components/table-advanced/TableAdvanced.ts (hunk 1 above)
Comment (helen-v) [reply web-components/src/components/table-advanced/TableAdvanced.ts:62]: Yes, thanks! It's my mistake. Will fix it.

**Comment by helen-v** [comment]
> Based on the ticket I see we need a more roving-tab-index solution like detailed in https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html
> Which is maybe a little different than what CAX-925 asks for. Difference being, should arrow keys navigate `up/down/left/right` to reach every cell (like that ARIA docs state) or just `up/down` like Shivam specifies on the ticket.
> 
> My feeling is align to the full ARIA nav capability instead of just row by row


I agree with you. Thank you for article, it's very helpful. Tomorrow, I will try implement it on our table.

File: web-components/src/components/table-advanced/scss/table-advanced.scss [hunk 2]
```diff
@@ -304,18 +304,13 @@ tbody {
...
       max-width: 0;
       padding: 0 calc(#{$advanced-table-main-indent} * 4);
       position: relative;
       text-align: left;
-    }
-
-    td:first-child,
-    th:first-child {
-      border-left: $advanced-table-main-border;
+      z-index: -1;
```
Comment (yananym) [inline web-components/src/components/table-advanced/scss/table-advanced.scss:313]:
```suggestion
```

**Review by yananym — APPROVED** [review]

**Pull request merged**